### PR TITLE
Feat: Added badge and button components

### PR DESCRIPTION
**Pull request by LordUttam**
Added badge and button components. Please review, thanks.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (Mohasin-Haque) [inline .vscode/settings.json:2]: add it in .gitignore

**Review by Mohasin-Haque — COMMENTED** [review]
Code is good but need some formatting

File: index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                    document.querySelectorAll('.header__logo img')[0].setAttribute('src', './assets/kalpa-white.png');
+                } else if (currentScrollPos > 30) {
+                    // going down
+                    document.querySelectorAll(".header--fixed")[0].classList.add('flip');
+                    document.querySelectorAll('.header__logo img')[0].setAttribute('src', './assets/kalpa-purple.png');
+                }
+                prevScrollPos = currentScrollPos;
+            }
+        }
+    </script>
```
Comment (ravikumar0403) [inline index.html]: You can extract this script into a separate js file.

File: pages/docs.css [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    font-weight: 600;
+    padding: .5rem 4px;
+}
+
+.main {
+    width: 80%;
+    margin: 1rem;
+    overflow-y: auto;
+    overflow-x: hidden;
+
```
Comment (ravikumar0403) [inline pages/docs.css]: here you can use the `overflow` shorthand property

File: pages/docs.css [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+.header--std {
+    display: flex;
+	align-items: center;
+    background-color: var(--base-color);
+    z-index: 5;
+    position: fixed;
+	width: 100%;
+	top: 0;
+    box-shadow: var(--secondary-box-shadow);
+	transition: top 0.2s ease-in-out;
```
Comment (ravikumar0403) [inline pages/docs.css]: check for the code formatting

File: pages/docs.css [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+
+.header--std ul li {
+	color: var(--primary-color);
+	font-size: 1.5rem;
+}
+
+
+.header--std ul li i {
+	color: var(--primary-color);
+}
```
Comment (ravikumar0403) [inline pages/docs.css]: Keep the indentation spacing consistent throughout the project

**Review by ravikumar0403 — COMMENTED** [review]
Left a few comments. 
and another suggestion would be to try to keep the PR small by splitting it based on features

**Pull request merged**